### PR TITLE
Add emotion support for SDK and Sample.

### PR DESCRIPTION
**Pull request by huxuan**
# Please do not merge this PR until the deployment of new version of Service as the changes are not compatible with previous version.

## TODOLIST
- [ ] Upload new version of nuget package.
- [x] Updatethe version in `packages.config` of Sample.
- [x] Update the `Reference` in Sample `csproj`.
- [ ] Update the `README.md`.

### CODE REVIEW

File: ClientLibrary/Contract/Emotion.cs [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+// OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+// WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+//
+
+namespace Microsoft.ProjectOxford.Face.Contract
+{
+    /// <summary>
+    /// Represents length of moustache, beard and sideburn
+    /// </summary>
+    public class Emotion
```
Comment (cthrash) [inline ClientLibrary/Contract/Emotion.cs]: Do we really want to duplicate this type from the emotion package?  It's FaceRectangle all over again @chbuehle, WDYT?

Comment (chbuehle) [inline ClientLibrary/Contract/Emotion.cs]: I agree we shouldn't duplicate the type.  What alternative do you suggest?

Comment (cthrash) [inline ClientLibrary/Contract/Emotion.cs]: Can we 

1. Move the entire implementation of Microsoft.ProjectOxford.Emotion.Contract.Emotion (and Score) to Microsoft.ProjectOxford.Common.Contract, and
2. Redefine as follows:
```
namspace Microsoft.ProjectOxford.Emotion.Contract
{
  public class Emotion : Microsoft.ProjectOxford.Common.Contract.Emotion
 {
   // class is empty, this type is defined for backward-compat only
  }
}
```
3. Have the Face API depend on the Common package,
4. Have the Face API use the Common.Contract.Emotion etc. types

It's possible I've overlooked some problem 😁

Comment (cthrash) [inline ClientLibrary/Contract/Emotion.cs]: + @chsienki

Comment (huxuan) [inline ClientLibrary/Contract/Emotion.cs]: Hi @cthrash & @chbuehle , thanks for your suggestion and I agree with your guys' opinions. I will look into it to see how to make the structure more elegant. AFAIK, there is no back-compatibility problem as emotion is a new feature. So adding `Emotion` contract in `Microsoft.ProjectOxford.Common` should be OK? Feel free to correct me if there is anything wrong.

Comment (cthrash) [inline ClientLibrary/Contract/Emotion.cs]: @chbuehle's team needs to own this moving forward.  He may very well delegate to @chsienki , but I'll leave that to @chbuehle.

Comment (chsienki) [inline ClientLibrary/Contract/Emotion.cs]: I'm happy to make the emotion changes, which we can reference from here. @chbuehle Any objections?

Comment (chbuehle) [inline ClientLibrary/Contract/Emotion.cs]: None from me, let's make the suggested changes.

Comment (huxuan) [inline ClientLibrary/Contract/Emotion.cs]: Thanks dude for your efforts. Please feel free to reach me when it is possible to reference the new emotion SDK.

**Comment by wangjun-microsoft** [comment]
signed off from my side.

File: ClientLibrary/Contract/Emotion.cs (hunk 1 above)
Comment (chsienki) [inline ClientLibrary/Contract/Emotion.cs]: The emotion types are now available in package Common 1.0.324 https://www.nuget.org/packages/Microsoft.ProjectOxford.Common/1.0.324

Comment (huxuan) [inline ClientLibrary/Contract/Emotion.cs]: @chsienki Thx for the notification. I will work on that before merge.

Comment (wangjun-microsoft) [inline ClientLibrary/Contract/Emotion.cs]: @chsienki ， @cthrash ，new [PR](https://github.com/Microsoft/Cognitive-Face-Windows/pull/21) has been submitted for referencing the package Common 1.0.324.

Comment (wangjun-microsoft) [inline ClientLibrary/Contract/Emotion.cs]: @chsienki ， @cthrash ，new [PR](https://github.com/Microsoft/Cognitive-Face-Windows/pull/21) has been submitted for referencing the package Common 1.0.324.

Comment (wangjun-microsoft) [inline ClientLibrary/Contract/Emotion.cs]: @chsienki ， @cthrash ，new [PR](https://github.com/Microsoft/Cognitive-Face-Windows/pull/21) has been submitted for referencing the package Common 1.0.324.

**Comment by huxuan** [comment]
Close as outdated.

**Pull request closed without merging**